app/onepassword_events_api/appserver/static/javascript/components/wizard.js

```diff
@@ -0,0 +1,179 @@
+import React from "react";
+
+const VERSION = "1.6.0";
+
+const e = React.createElement;
+export default class Wizard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentStep: 1,
+      totalSteps: props.steps.length,
+      success: false,
+      error: "",
+      authToken: "",
+    };
+  }
+
+  handleNext = async (event) => {
+    event.preventDefault();
```

Comment:
we shouldn't need to call preventDefault every time we create a onClick handler, usually that is only used with  buttons that submit. Is the page reloading when you remove this?